pom.xml

```diff
@@ -50,9 +51,9 @@
 
     <scm>
         <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
-        <developerConnection>scm:git:https://github.com/${gitHubRepo}.git</developerConnection>
+        <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
         <url>https://github.com/${gitHubRepo}</url>
-        <tag>v22.2.39</tag>
+        <tag>${scmTag}</tag>
```

Comment:
As per latest recommendations in the archetype: https://github.com/jenkinsci/archetypes